### DISCUSSION TITLE
Adjust fps of video broadcast

### DISCUSSION THREAD

**Discussion by darld** [TannerGabriel/WebRTC-Video-Broadcast#50] [Q&A] [answered]
Hi Gabriel, 

Can you suggest me the best way to adjust the fps of video steaming? I have tried to change code in broadcast.js, but the streaming is very lag and freeze sometime.

**Comment by TannerGabriel** [comment, marked as answer]
Sorry I missed this question. I'm not sure that the lag you are talking about has something to do with the framerate. That said, you can change the framerate using the `applyConstrains` parameter ([example](https://stackoverflow.com/questions/29302617/control-video-send-framerate-on-the-fly-in-webrtc))

**Reply by darld** [reply]
thanks @TannerGabriel That's work.
